tests/playwright/page-object-model/trust/contacts-page.ts

```diff
@@ -25,18 +40,25 @@ class ContactsPageAssertions extends BaseTrustPageAssertions {
   }
 
   async toSeeCorrectDfeContacts (): Promise<void> {
-    await expect(this.contactsPage.dfeContactsCard).toContainText('Keyshawn Hermiston')
-    await expect(this.contactsPage.dfeContactsCard).toContainText('Keyshawn.Hermiston@education.gov.uk')
-    await expect(this.contactsPage.dfeContactsCard).toContainText('Ayana Lueilwitz')
-    await expect(this.contactsPage.dfeContactsCard).toContainText('Ayana.Lueilwitz@education.gov.uk')
+    await expect(this.contactsPage.dfeContactsCard).toContainText(`Trust relationship manager ${this.contactsPage.currentTrust.trustRelationshipManager.fullName} ${this.contactsPage.currentTrust.trustRelationshipManager.email}`)
+    await expect(this.contactsPage.dfeContactsCard).toContainText(`SFSO (Schools financial support and oversight) lead ${this.contactsPage.currentTrust.sfsoLead.fullName} ${this.contactsPage.currentTrust.sfsoLead.email}`)
   }
 
   async toSeeCorrectTrustContacts (): Promise<void> {
-    await expect(this.contactsPage.trustContactsCard).toContainText('Tyler Welch')
-    await expect(this.contactsPage.trustContactsCard).toContainText('Tyler.Welch@abbeylaneacademiestrust.co.uk')
-    await expect(this.contactsPage.trustContactsCard).toContainText('Courtney Pacocha')
-    await expect(this.contactsPage.trustContactsCard).toContainText('Courtney.Pacocha@abbeylaneacademiestrust.co.uk')
-    await expect(this.contactsPage.trustContactsCard).toContainText('Lowell Hoppe')
-    await expect(this.contactsPage.trustContactsCard).toContainText('Lowell.Hoppe@abbeylaneacademiestrust.co.uk')
+    const requiredTrustContacts = ['Accounting Officer', 'Chair of Trustees', 'Chief Financial Officer']
+    const trustContacts = this.contactsPage.currentTrust.governors.filter(x => requiredTrustContacts.some(n => n === x.role))
+    await expect(this.contactsPage.trustContactsCard).toContainText(`Chair of trustees ${trustContacts[0].fullName} ${trustContacts[0].email}`)
```

Comment:
Unfortunately we can't guarantee that this approach (direct indexing of `trustContacts`) will always work:

- The data might have an old/historic governor in - the contacts *should* only show current governors
- The governor order may change in the future - e.g. after a regeneration of `trusts.json` the accounting officer might be the first contact